### ISSUE TITLE
[Translate] Degree of interpretation for translated content

### ISSUE THREAD

**Issue by ann0see** [jamulussoftware/jamulussoftware.github.io#146]
This issue should discuss how far interpretation of the English version should go in translations. 

For me I think these main points have to be considered:

## What is the translation for?

* A non English speaking person should understand the same as if he read the English version and think the text was written in his native language. 

For me this means that:

1. The translation should be based on the English version and should not add more information (if you want to, add it to the English version)
2. The translated version should not sound like an English text in the language we translate it to. This means that terms like "headless" can't be translated into something like "without head". The text should sound like it was written in the language we translate it to. Therefore although the content is similar, we do not need to use the exact same words. 
3. Concerning guides, links, ... to pages in the language we translate to: I'd strongly advise to link to pages in this language, not English if the quality is ok.

**Comment by gegeweb** [comment]
I agree, but some questions:

Point 1, what doing if when translating we notice that some formulations are not clear, or that there is an error? Or missing important point? Obviously the (my) reflex is to correct the error in the translated text. It should be possible to submit both, the translation and the corrected original.
Sometimes, adding something could be useful for the translated language but not in other language, as example, the note about the word « gigue » in the referenced PR.

Point 2, totally agree, especialy for the "Software Manual" page where we describe the UI and where we explain what do the elements of the UI.
Try to keep closed to the original wording with a litteral translation is not (IMHO) the good approach. We should keep the original meaning. And sometimes staying too close to English (which is, if I'm not mistaken, often a translation from German) distorts the meaning.
Sometimes too, a word, or an expression doesn't make sens for everybody, for example, see https://github.com/jamulussoftware/jamuluswebsite/pull/47.
"Jam" -> "bœuf" or "Jamming" -> "bœuffer", is not a wrong translation, French musicians use idioms like « taper le bœuf », « faire un bœuf » (but never the verb, wich is not in the dictionnary) but doesn't make sens for @Saimyx as you can see. While "improviser" is clearer to him.
Another example for the French version (doc and UI, see https://github.com/corrados/jamulus/issues/759) the use of "chariot" for "fader", it is not wrong but doesn't make sens in the context of a mixing desk or console.
We usualy use "potentiomètre" or "canal" to talk about the fader or the input channel.
Last example « Compile this bad boy », trying to find an equivalent causes an incongruous result. It is preferable to use a more neutral and common formulation. In this case "Compiler le logiciel" or "Compiler Jamulus".

Point 3, I totally agree, of course! ;)

**Comment by gegeweb** [comment]
Another Idea, as the note about "gigue", Having a glossary of the terms and words used to clarify them (if the English word is retained, or if it is ambiguous) for the language concerned might be good.

Your opinion?

**Comment by ignotus666** [comment]
> Point 1, what doing if when translating we notice that some formulations are not clear, or that there is an error? Or missing important point? Obviously the (my) reflex is to correct the error in the translated text. It should be possible to submit both, the translation and the corrected original.

If it's an error and you are confident of how to correct it, submit a PR to 'release', and if unsure, open an issue so someone else can have a look at it. The same if you consider a sentence should be reworded to make it clearer. But if you think it's missing something and this should also be included in other translations, first submit your proposal to 'changes'.

When I see mistakes or awkward wordings I don't translate them as-is either and try to correct these things in my translation; as long as you're not adding anything new I think it's the right way to go - it would be absurd to carry them over. Up until now I haven't bothered to correct the English version as I think it's best to wait for the new content to take shape, to avoid wasting time correcting things that might get changed anyway.

**Comment by trebmuh** [comment]
I agree with the 3 points of @ann0see's first message.

> Point 1, what doing if when translating we notice that some formulations are not clear, or that there is an error? Or missing important point? 

Then, it needs discussion.


> Obviously the (my) reflex is to correct the error in the translated text.

This is a bad way to do, because you could make a mistake. That's why you have to discuss it first.


> It should be possible to submit both, the translation and the corrected original.

Only after a discussion.


> Sometimes, adding something could be useful for the translated language but not in other language, as example, the note about the word « gigue » in the referenced PR.

Agreed but it needs to have a very good reason for that (like for "gigue").




> Point 2, totally agree, especialy for the "Software Manual" page where we describe the UI and where we explain what do the elements of the UI.
> Try to keep closed to the original wording with a litteral translation is not (IMHO) the good approach.

Unless there is a good reason, we should keep it close to the original text. Obviously, there are some times when a translation needs to be adaptative. That said, to be honest, I've see too many times translation where you are changing things not because it's needed, but because "you feel it's better like that". See [this comment of mine](https://github.com/jamulussoftware/jamuluswebsite/pull/106#issuecomment-739028298) about that and why you should keep your translations closer to English. Please, note that I didn't say "at all cost" or "only litteral", but, "stay closer". When it's needed, then no worries, even if we need to keep an English word every now and then (think "ping time" as a recent example).


>  We should keep the original meaning.

There is no opposition between "keeping the original meaning" and "staying the closest we can to the English text".


> And sometimes staying too close to English (which is, if I'm not mistaken, often a translation from German) distorts the meaning.

It is true that this can happen, but not very often given the content of the texts we translate here. It is not poetry, we're mainly translating technical information and documentation.



> Sometimes too, a word, or an expression doesn't make sens for everybody, for example, see #47.
> "Jam" -> "bœuf" or "Jamming" -> "bœuffer", is not a wrong translation, French musicians use idioms like « taper le bœuf », « faire un bœuf »

That is not right. As a musician since I'm 16 (I'm 40 now), I can tell that I've heard and used this word all around.


> but never the verb, wich is not in the dictionnary

That isn't completely right. See [the reverso dictionary](https://dictionnaire.reverso.net/francais-definition/boeuffer), [l'obs dictionary](https://la-conjugaison.nouvelobs.com/definition/boeuffer.php), [the wikipedia dictionary](https://fr.wiktionary.org/wiki/b%C5%93uffer).

(BTW, there is only one "n" to "dictionary" in English)


> but doesn't make sens for @Saimyx as you can see. While "improviser" is clearer to him.

It could be used as a drop in replacement as well in some cases. But to be honest, I'm not fond of this because "improviser" sounds more like a "technical soloing person" meaning than a "jamming group of people" meaning. In this case (to translate "jam") we have 3 propositions:
* "improviser/impro" which sounds like a unique technical soloing person (the soloing person is doing the impro, the others are accompanying)
* "jam" which is an anglicism (to be avoided if possible)
* "bœuf/bœuffer" which is known and used

So the best choice here is the bœuf/bœuffer one.


> Another example for the French version (doc and UI, see corrados/jamulus#759) the use of "chariot" for "fader", it is not wrong but doesn't make sens in the context of a mixing desk or console.

Sorry but it does. I'm using this translation in quite a lot of audio software I'm translating and so far, I've got not criticism. Moreover I've explained you carefully that it's a trade-off [in this comment](https://github.com/corrados/jamulus/issues/759#issuecomment-738912075) and [in this comment, what are the different suggestions](https://github.com/corrados/jamulus/issues/759#issuecomment-738837442).


>  We usualy use "potentiomètre" or "canal" to talk about the fader or the input channel.

That is not right. As explained [here](https://github.com/jamulussoftware/jamuluswebsite/pull/106#issuecomment-739028298) this is how an **electronic person** or a **sound technician** is talking about it. But such a person is using a lingo/jargon/slang which isn't catering for all the people that a translation should cater for.


> Last example « Compile this bad boy », trying to find an equivalent causes an incongruous result. It is preferable to use a more neutral and common formulation. In this case "Compiler le logiciel" or "Compiler Jamulus".

Disagree here. We found a good one. "Compiler ce vilain garçon".



So now that I answered point by point, I feel I need to tell you the following. It is very time-consuming that you're repeating the same things in different places. Please keep one discussion in one place. As an example, today, I had to answer in 3 different places about the "_chariot_" translation. To begin with, you didn't use it in the documentation translation while it's the word used in the GUI. I told it to you when doing the review. Then you went to the jamulus software too where I had to answer too, then we are redoing the same here. I'm afraid that this is very looking like you don't like this translation trade-off choice made in the GUI, and you're just not accepting that it's a compromise.

Please, bear in mind that when I'm doing a review of the translation PR you're doing, I'm doing a **lot** of compromises because there are a **lot** of things that I wouldn't have translated the same way as you do. But I do accept not to be egotistic because this is a shared work. Please, do the same, understand the needed compromises and trade-off. This is the price to pay for working together on a translation. I think that I'm nice enough and the French PR are mostly from your writings. But even with me compromising a lot, still it sometimes looks like that it's too much for you. That is sad because I think we made some progress on the last translation we did. Don't we?

**Comment by gegeweb** [comment]
> So now that I answered point by point, I feel I need to tell you the following. It is very time-consuming that you're repeating the same things in different places. Please keep one discussion in one place. As an example, today, I had to answer in 3 different places about the "_chariot_" translation.

Perhaps because (for this page) no reply to the comment on the originaly PR: https://github.com/jamulussoftware/jamuluswebsite/pull/71#issuecomment-734784019?

> That is sad because I think we made some progress on the last translation we did. Don't we?

Yes we did.

> Disagree here. We found a good one. "Compiler ce vilain garçon".

Yes but always incongruous for me. ;)

**Comment by gegeweb** [comment]
> Then you went to the jamulus software too where I had to answer too, then we are redoing the same here. I'm afraid that this is very looking like you don't like this translation trade-off choice made in the GUI, and you're just not accepting that it's a compromise.

I went to the Jamulus Software repo to open an issue, because when working on the translation of the 'Software-Manual' I have take care about the UI and found something sounded strange to me. Especially about the sorting channels feature.
I didn't understand some of the wordings.
And yes, to make suggetions. I'm a end user of Jamulus Software, I really like this Software. And If I can contribute en try to improve something, I do it. Jamulus is a Free Software.

Yes I didn't feel happy with "chariot", yes I work in the corporation of sound technicians. So yes, I usualy use "canal" to talk about a channel that corresponding at a "slice" of the mixing desk input buses section and "potentiomètres" for talk about faders (yes it come from the electronic component). That is usual. And why not use it and if needed explain the meaning of this term ine a glossary (for the doc)?
And I've explain this choice with a comment on the first PR (closed and reopened in a new because the freeze and merge of `translation` branch. 

In fact, it seems I was right about the misunderstanting in the French UI wich come from a wrong German to English translation. The English having been used as the reference.

Like me, you've make an (your) interpretation of the english wordings and then try to translate as better as possible this in the French UI. With the mistake of the english translation.
When I've looked at the english wording, before checking the German, Spanish and Italian, I didn't take care about the wrong usage of the plural, and my brain have, by reflex, translated "channel users" to "user channels", which was the intended meaning. And especially because that's what functionality does.

I know perfectly well the problem of describing a user interface in a documentation. I have to do it in my work. And I have recently had to do it for non-French speaking users while waiting for the interface of our software in French to be translated into English. And I had to translate all my docs into English for these new users.
And above all, I am careful to bring myself up to the level of the users, while trying to do pedagogy by sometimes describing the technical aspects when I think it is usefull for understanding some behaviors.

To go back in the initial topic, I'm like @ignotus666 and _when I see mistakes or awkward wordings I don't translate them as-is either and try to correct these things in my translation_. That's the meaning of "keep close to the EN is not (IMHO) the good approach". Especially knowing that English is a translation of German most of the time, therefore may contain errors.
That's why for some pages I also looked at the Spanish translation, a language I understand better than German (even if I have some remains from my 4 years learning German in school).
